tools/testsys-config/src/lib.rs

```diff
@@ -261,6 +261,14 @@ pub struct GenericVariantConfig {
     pub control_plane_endpoint: Option<String>,
     /// The path to userdata that should be used for Bottlerocket launch
     pub userdata: Option<String>,
+    /// The directory containing Bottlerocket images. For metal, this is the directory containing
+    /// gzipped images.
+    pub os_image_dir: Option<String>,
```

Comment:
If this is really specific to gzip or metal images, should it be called gzip_image_dir or metal_image_dir?

Reply:
Currently it is for metal, but it doesn't have to be. A custom test agent for eks may need to use the `os_image_dir`.